docs/book/src/roadmap.md

```diff
@@ -1,52 +1,56 @@
+|UX, Clusterctl|Explore alternatives to rate limited GitHub API for clusterctl|[#3982](https://github.com/kubernetes-sigs/cluster-api/issues/3982)|
+|API, Contracts|Inconsistent provider behaviour with API server ports|[#5517](https://github.com/kubernetes-sigs/cluster-api/issues/5517)|
+|API, Core Improvements|Represent Failure domains availability|[#4031](https://github.com/kubernetes-sigs/cluster-api/issues/4031)|
+|Core Improvements|Support syncing Node labels |[#493](https://github.com/kubernetes-sigs/cluster-api/issues/493)|
+|Core Improvements|Drain workload clusters of service Type=Loadbalancer on teardown|[#3075](https://github.com/kubernetes-sigs/cluster-api/issues/3075)|
+|Core Improvements|Bubble up conditions for worker nodes|[#3486](https://github.com/kubernetes-sigs/cluster-api/issues/3486)|
+|Core Improvements|Support multiple kubeconfigs for a provider|[#3661](https://github.com/kubernetes-sigs/cluster-api/issues/3661)|
+|Core Improvements|Block on unsupported Kubernetes version|[#4321](https://github.com/kubernetes-sigs/cluster-api/issues/4321)|
+|Core Improvements|Improve certificate management in Cluster API|[#5490](https://github.com/kubernetes-sigs/cluster-api/issues/5490)|
+|Core Improvements|Add support for RolloutAfter to MachineDeployments|[#4536](https://github.com/kubernetes-sigs/cluster-api/issues/4536)|
+|Core Improvements|Add support for RolloutAfter to ClusterClass|[#5218](https://github.com/kubernetes-sigs/cluster-api/issues/5218)|
+|Core Improvements|MHC should provide support for checking Machine conditions|[#5450](https://github.com/kubernetes-sigs/cluster-api/issues/5450)|
+|Core Improvements|MachinePool workers support in ClusterClass|[#5991](https://github.com/kubernetes-sigs/cluster-api/issues/5991)|
+|Core Improvements|Allow using a ClusterClass across namespaces|[#5673](https://github.com/kubernetes-sigs/cluster-api/issues/5673)|
+|Extensibility|Introduce the runtime SDK|[#6181](https://github.com/kubernetes-sigs/cluster-api/pull/6181)|
+|Extensibility|Introduce topology mutation hooks| |
+|Extensibility|Introduce lifecycle hooks for enabling addon orchestration|[#5491](https://github.com/kubernetes-sigs/cluster-api/issues/5491)|
+|Extensibility|Continuous ClusterResourceSetStrategy|[#4807](https://github.com/kubernetes-sigs/cluster-api/issues/4807)|
+|Visibility|Integrate with the Cluster API metrics project| |
+|Visibility|Improve CAPI logs and ensure consistent with updated K8s guidelines| |
+
+For a detailed view of all the issues in the milestone see [milestone v1.2](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.2). 
 
 ---
 
 ## Backlog
 
 > Items within this category have been identified as potential candidates for the project
-> and can be moved up into a milestone if there is enough interest.
+> and can be moved up into a milestone if there is enough interest/volounteers for implementation.
```

Comment:
lgtm but I wonder of the value of trying to keep this updated in the book, maybe the link to next it's enough.

Reply:
That's fair, I will drop the list of issues for the backlog

Reply:
I wonder if we're able to keep the list of issues for the v1.2 milestone up-to-date in the book / if it's worth the effort.

Maybe for v1.2 some label combination that we can link to would be better too.

Reply:
Ah nvm. I assume the v1.2 list is a curated list with rather "high-level" issues compared to all issues that we have?

Reply:
That being said, this PR is an improvement in any case, so no objection to merge this from my side.